<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main">
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0,0">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QRadioButton" name="BT_old_version">
             <property name="font">
              <font>
               <family>Gabriola</family>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>기존버전</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="BT_block_version">
             <property name="font">
              <font>
               <family>Gabriola</family>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>블록버전</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BT_F5">
             <property name="font">
              <font>
               <family>Gabriola</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>새로고침</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1,1">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>계정</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>키 개수</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>사용자</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>유형</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>메모</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="BT_search">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>검색</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BT_Addpage">
          <property name="text">
           <string>사용자 추가</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="minimumSize">
         <size>
          <width>663</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">text-align: center;</string>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>계정</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>키 개수</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>소유자</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>유형</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>메모</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>사용중</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="text2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
        <item row="0" column="2">
         <widget class="QLineEdit" name="LE_account">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="CB_category">
          <item>
           <property name="text">
            <string>구매</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>무상</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>임시</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>계정</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>키 개수</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>소유자</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>계정 유형</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="SB_keyCount"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>   메모</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="LE_owner"/>
        </item>
        <item row="4" column="2">
         <widget class="QTextEdit" name="TE_memo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="modify_BT">
          <property name="text">
           <string>수정</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="text">
           <string>삭제</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BT_useclaer">
          <property name="text">
           <string>사용자 초기화</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
